src/extensions/registries/index.ts

```diff
@@ -10,3 +10,4 @@ export * from "./page-menu-registry";
 export * from "./entity-setting-registry";
 export * from "./catalog-entity-detail-registry";
 export * from "./protocol-handler";
+export * from "./modal-registry";
```

Comment:
This will export the injection token, which isn't usable yet in the extension API. Should we instead just export the `ClusterModalRegistration` type?

Reply:
Removed injection token export from `./modal-registry`